thrift.js

```diff
@@ -114,6 +114,7 @@ function Thrift(options) {
     this.surface = this;
 
     this.linked = false;
+    this.releaseSources = options.releaseSources;
```

Comment:
document what this does?

Reply:
Not against of adding it but do you think this is worth the effort? Library is in maintenance mode and unfortunately has barely any documentation for it's interfaces and seeding one at the end of its lifetime seems vain effort.